Xamarin.Forms.Platform.Android/AppCompat/FrameRenderer.cs

```diff
@@ -83,6 +83,13 @@ bool IOnTouchListener.OnTouch(AView v, MotionEvent e)
 					ScaleGestureDetectorCompat.SetQuickScaleEnabled(_scaleDetector.Value, true);
 				handled = _scaleDetector.Value.OnTouchEvent(e);
 			}
+
+			if (_gestureDetector.IsValueCreated && _gestureDetector.Value.Handle == IntPtr.Zero)
+			{
+				// This gesture detector has already been disposed, probably because it's on a cell which is going away
```

Comment:
The situation being addressed is a gesture detector which has been disposed on the Java side because the list view cell which contains it has been collected - see the description from the [original PR](https://github.com/xamarin/Xamarin.Forms/pull/706).